README.md

```diff
@@ -5,7 +5,7 @@ A simple app that listens on a port and proxies (and possibly fans out) data to
 ## Usage
 
 ```
-$ mix socket_proxy --listen-port 8001 12.12.12.12:8002 13.13.13.13:8003
+$ env SOCKET_PROXY_LISTEN_PORT=8001 SOCKET_PROXY_DESTINATIONS="12.12.12.12:8002, 13.13.13.13:8003" mix socket_proxy --no-halt
```

Comment:
Hmm, I'm still kind of confused by this new setup. So there's no mix task called `socket_proxy` anymore? It's just starting up the application? Would the same thing happen if you did `mix run` instead of `mix socket_proxy`?

Reply:
Oops, good catch here! This should be `mix run`. `mix socket_proxy` doesn't work anymore since I removed that mix task.